src/lib/annotations/doc/DocAnnotator.js

```diff
@@ -445,11 +445,12 @@ class DocAnnotator extends Annotator {
 
         this.annotatedElement.removeEventListener('mouseup', this.highlightMouseupHandler);
 
-        if (this.annotationService.canAnnotate) {
+        if (this.canAnnotate) {
             this.annotatedElement.removeEventListener('dblclick', this.highlightMouseupHandler);
             this.annotatedElement.removeEventListener('mousedown', this.highlightMousedownHandler);
             this.annotatedElement.removeEventListener('contextmenu', this.highlightMousedownHandler);
-            this.annotatedElement.removeEventListener('mousemove', this.getHighlightMouseMoveHandler());
+            this.annotatedElement.removeEventListener('mousemove', this.getHighlightMouseMoveHandler);
+            this.highlightMousedownHandler = null;
```

Comment:
And also this needed to be `this.highlightMousemoveHandler = null` not `this.highlightMouseDownHandler = null`